nautobot/extras/templates/extras/jobqueue_retrieve.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block content_left_page %}
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <strong>Job Queue</strong>
+            </div>
+
+            <table class="table table-hover panel-body attr-table">
+                <tr>
+                    <td>Name</td>
+                    <td>
+                        <span>{{ object.name }}</span>
```

Comment:
Don't need the `<span>...</span>` wrappings here and below unless I'm missing something?